docs/manual-EN/2.query-language/3.language-structure/pipe-syntax.md

```diff
@@ -10,13 +10,13 @@ Meanwhile, nGQL uses shell style `PIPE (|)`.
 ```ngql
 GO FROM 201 OVER edge_serve | GO FROM $-.id OVER edge_fans | GO FROM $-.id ...
 
-GO FROM 100 OVER like YIELD like._dst AS dstid, $$.player.name AS Name  | GO FROM $-.dstid OVER like YIELD like._dst, like.likeness, $-.Name
+GO FROM 100 OVER follow YIELD follow._dst AS dstid, $$.player.name AS Name  | GO FROM $-.dstid OVER follow YIELD follow._dst, follow.likeness, $-.Name
```

Comment:
I do not know whether you will keep the word "likeness". Please confirm it.

Reply:
likeness > grade
Will open a new pr to take care of all of the `like`s.